lib/slack-profile-cache.js

```diff
@@ -251,3 +252,16 @@ function timeout (ms) {
 async function sleep (timeoutMs) {
   await timeout(timeoutMs)
 }
+
+async function getUsersList (users, cursor) {
+  return slackWebClient.users.list({limit: 200, cursor: cursor}).then(
```

Comment:
I see that the `getUsersList` function is already marked `async` - why not replace `then`/`catch` with `try`/`await`/`catch`?

Reply:
ok, I'll do it!